### DISCUSSION TITLE
influxdb 2.x steep upgrade path, shouldn't there be a version@1.8 tag?

### DISCUSSION THREAD

**Discussion by jum** [Homebrew/discussions#1110] [Casks]
The upgrade to influxdb 2.x is rather complicated. Due to an brew update I do not get the old version back. Shouldn't there be a way to install the @1.8.x version meanwhile? There is some time needed to do the migration.

**Comment by carlocab** [comment]
Maybe. This is in progress at https://github.com/Homebrew/homebrew-core/pull/73680.

However, if there weren't already an open PR, the best way to answer your question would've been to open one.
